lightning/src/routing/router.rs

```diff
@@ -719,8 +727,9 @@ where L::Target: Logger {
 			node_info.features.supports_basic_mpp()
 		} else { false }
 	} else { false };
-	log_trace!(logger, "Searching for a route from payer {} to payee {} {} MPP", our_node_pubkey,
-		payment_params.payee_pubkey, if allow_mpp { "with" } else { "without" });
+	log_trace!(logger, "Searching for a route from payer {} to payee {} {} MPP and {} first hops {}overriding the network graph", our_node_pubkey,
+		payment_params.payee_pubkey, if allow_mpp { "with" } else { "without" },
+		first_hops.map(|hops| hops.len()).unwrap_or(0), if first_hops.is_some() { "" } else { "not " });
```

Comment:
Looks like an inadvertent change / merge resolution?

Reply:
Ow, thank you for pointing this out! Fixed with https://github.com/lightningdevkit/rust-lightning/commit/386a0b553130173f576ecc89424fcd6791a41dcc.